docs/nidcpower/class.rst

```diff
@@ -129,7 +129,7 @@ abort
     .. py:method:: abort()
 
             Transitions the NI-DCPower session from the Running state to the
-            Committed state. If a sequence is running, it is stopped. Any
+            Uncommitted state. If a sequence is running, it is stopped. Any
```

Comment:
Weird, I didn't know we had an "uncommitted" state. Where did this get pulled from?

Reply:
From their internal metadata